.env

```
@@ -50,6 +50,7 @@ VITE_FEATURE_NEW_WALLET_FLOW=true
 VITE_FEATURE_NEW_LIMIT_FLOW=true
 VITE_FEATURE_THORCHAIN_SWAPPER_ACK=false
 VITE_FEATURE_ACTION_CENTER=true
+VITE_FEATURE_QUICK_BUY=true
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The VITE_FEATURE_QUICK_BUY key should go before the VITE_FEATURE_RFOX key

(UnorderedKey)